stream-chat-android-compose-sample/src/main/java/io/getstream/chat/android/compose/sample/ui/MessagesActivity.kt

```diff
@@ -131,17 +131,18 @@ class MessagesActivity : BaseConnectedActivity() {
                         composerViewModel.setMessageMode(MessageMode.MessageThread(message))
                         listViewModel.openMessageThread(message)
                     },
-                    onImagePreviewResult = { result ->
+                    // TODO edit docs
```

Comment:
have we resolved this TODO or do we still have work? @MarinTolic

Reply:
Yeah, this was already done, I just forgot to remove the `TODO`.

Next time I'll put a `TODO` to remove the `TODO` to make sure I don't leave any unnecessary ones around.